tasks/gitlab_helpers.py

```diff
@@ -55,31 +57,43 @@ def create_gitlab_annotations_report(ci_job_id: str, ci_job_name: str):
             {
                 "external_link": {
                     "label": "CI Visibility: This job instance",
-                    "url": get_link_to_job_id(ci_job_id),
+                    "url": get_link_to_pipeline_job_id(ci_job_id),
+                }
+            },
+            {
+                "external_link": {
+                    "label": "CI Visibility: This job test runs",
```

Comment:
```suggestion
                    "label": "Test Visibility: This job test runs",
```
So that we stick to how it's named in the UI, wdyt?
![image](https://github.com/DataDog/datadog-agent/assets/24426034/160a8e1f-1b0f-4853-af31-40a390a047af)